kafka-ui-react-app/src/components/Topics/New/New.tsx

```diff
@@ -29,6 +29,15 @@ const New: React.FC = () => {
   const history = useHistory();
   const dispatch = useDispatch();
 
+  const { search } = useLocation();
+  const name = new URLSearchParams(search).get('name') || '';
+  const partitionCount = new URLSearchParams(search).get('partitionCount') || 1;
+  const replicationFactor =
+    new URLSearchParams(search).get('replicationFactor') || 1;
+  const inSyncReplicas = new URLSearchParams(search).get('inSyncReplicas') || 1;
+  const cleanUpPolicy =
+    new URLSearchParams(search).get('cleanUpPolicy') || 'Delete';
```

Comment:
Can we store `new URLSearchParams(search)` in a variable and reuse it? Please, check if we can, that way we'll prevent duplication.

Reply:
done